mne/io/constants.py

```diff
@@ -136,10 +136,11 @@ def __setattr__(self, attr, val):
 FIFF.FIFF_DESCRIPTION    = FIFF.FIFF_COMMENT # (Textual) Description of an object
 FIFF.FIFF_DIG_STRING     = 234          # String of digitized points
 FIFF.FIFF_LINE_FREQ      = 235    # Line frequency
-FIFF.FIFF_CUSTOM_REF     = 236    # Whether a custom reference was applied to the data
+FIFF.FIFF_CUSTOM_REF     = 236    # Whether a custom reference was applied to the data (whoops, overlaps...)
```

Comment:
We need to be more careful choosing new FIFF constants. The custom ref one
we chose was already occupied by the hpi freq (both are now 236)